### DISCUSSION TITLE
Can @nxext/stencil get an update to the lastes stencil core 4.3.0?

### DISCUSSION THREAD

**Discussion by silvan88** [nxext/nx-extensions#1037] [Q&A]
We are interested in integrating Stencil into our Nx workspace. However, we've noticed that the @nxext/stencil package is currently using @stencil/core version 2.17.1, whereas the latest Stencil release is 4.30. Would it be possible to update the @stencil/core to the latest version?

**Comment by Joeylaan** [comment]
We are also running into this problem for our project. Any updates about this?

**Comment by DominikPieper** [comment]
The v17.0.1 supports Stencil 4.7
